lib/sandbox/execute.js

```diff
@@ -53,13 +55,21 @@ module.exports = function (bridge, glob) {
             abortEventName = EXECUTION_ABORT_EVENT_BASE + id,
             responseEventName = EXECUTION_RESPONSE_EVENT_BASE + id,
             cookiesEventName = EXECUTION_COOKIES_EVENT_BASE + id,
+            assertionEventName = EXECUTION_ASSERTION_EVENT_BASE + id,
 
             // extract the code from event. The event can be the code itself and we know that if the event is of type
             // string.
             code = _.isFunction(event.script && event.script.toSource) && event.script.toSource(),
             // create the execution object
             execution = new Execution(id, event, context, options),
 
+            dispatchAssertions = function (assertions) {
```

Comment:
@codenirvana can you extensively comment above this function? This is defined a bit far away from its actual usage, hence a leading context would help.